internal/provider/baremetal/inventory.go

```diff
@@ -13,6 +16,19 @@ func (p *baremetalProvider) CleanPlatformValuesFromDBUpdates(_ map[string]interf
 	return nil
 }
 
+func (p *baremetalProvider) GetActualSchedulableMasters(cluster *common.Cluster) (bool, error) {
```

Comment:
Don't mean to nitpick on naming but, why do you think we need `Actual` in the name of this function? Is there a way to get a list of schedulable masters that are not really schedulable?

Reply:
"Actual" in this function name means "Get the actual value of the scheduling property" because it's not always the value contained in the database.